examples/resources/ybm_integration/vm-integration.tf

```diff
@@ -0,0 +1,7 @@
+resource "ybm_integration" "victoriametrics" {
```

Comment:
Just like "sumologic" we can name this file as "victoriametrics-integrations.tf"
"vm" is slightly confusing